curriculum/challenges/spanish/03-front-end-libraries/sass/nest-css-with-sass.spanish.md

```diff
@@ -10,7 +10,34 @@ localeTitle: Nest CSS con Sass
 ---
 
 ## Description
-<section id="description"> Sass permite <code>nesting</code> reglas CSS, que es una forma útil de organizar una hoja de estilo. Normalmente, cada elemento está enfocado en una línea diferente para diseñarlo, así: <blockquote> nav { <br> color de fondo: rojo; <br> } <br><br> nav ul { <br> estilo de lista: ninguno; <br> } <br><br> nav ul li { <br> pantalla: bloque en línea; <br> } </blockquote> Para un proyecto grande, el archivo CSS tendrá muchas líneas y reglas. Aquí es donde el <code>nesting</code> puede ayudar a organizar su código al colocar reglas de estilo secundarias dentro de los elementos principales respectivos: <blockquote> nav { <br> color de fondo: rojo; <br><br> ul { <br> estilo de lista: ninguno; <br><br> li { <br> pantalla: bloque en línea; <br> } <br> } <br> } <br></blockquote></section>
+<section id="description"> Sass permite anidamiento (<code>nesting</code>) de reglas CSS, que es una forma útil de organizar una hoja de estilo. Normalmente, cada elemento se escribe en una línea diferente para darle estilo así: 
+  
+```html
+  nav {
+    background-color: red;
+  } 
+  
+  nav ul { 
+   list-style: none;
+  }
+
+  nav ul li {
+    display: inline-block;
+  } 
+ ```
+ Para un proyecto grande, el archivo CSS tendrá muchas líneas y reglas. Aquí es donde el <code>nesting</code> puede ayudar a organizar su código al colocar reglas de estilo secundarias dentro de los respectivos elementos principales: 
+ 
+```html
+  nav {
+    background-color: red;
+    ul {
+      list-style: none;
+      li {
+        display: inline-block;
+      }
+    }
+  }
+```
 
```

Comment:
I believe we need `</section>` here. Will have to check once local setups are working again.

Reply:
Good point, I agree.

Reply:
I corrected this for the user.